home and about page

hopeforkoami opened the pull request:
in this branch i implement the following features:
-build the home and the about pages 
- each of them has a mobile and desktop version
- The home page have a link in the menu to the about page
- The logo in the header links to the home page.
- When the user clicks the hamburger button on the header, the mobile menu appears over the page.
- The mobile menu has a close (X) button that closes the menu.
- The section "Featured speakers" is created dynamically in JavaScript

hopeforkoami committed:
error correction

hopeforkoami committed:
linter errors correction

fobadara commented on style.css:
```diff
@@ -1,6 +1,6 @@
 @import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
 
-@media screen and (max-width:767px) {
+@media screen and (max-width: 767px) {
```
- [ ] I think a major cause of the overflow problem. is that you didn't use a [mobile-first](https://zellwk.com/blog/how-to-write-mobile-first-css/) approach. It is a requirement for you to follow the [mobile-first](https://zellwk.com/blog/how-to-write-mobile-first-css/) approach [REQUIRED]

fobadara reviewed (changes requested):
Hi @hopeforkoami , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Linters are passing
✅ Descriptive pull request

## Required Changes ♻️
_Check the comments under the review._
I included some comments here because there is no way for me to write inline comments since the affected lines of text/code are missing.
- [ ] Kindy fix the link to your live demo because it is currently throwing a 404 error. [REQUIRED]
- [ ] I can't find the link to the video description of your app. [REQUIRED]
- [ ] In your acknowledgements section in your README you are not crediting Cindy Shin. Kindly fix that [REQUIRED]
- [ ] The entire page for your index and about page is overflowing which is disturbing the layout of it's content. Kindly find and  fix  the cause of this overflow issue [REQUIRED]
- [ ] Kindly fix the following errors in the home page as shown in your design below: [All are Required]
 - The navigation links are overflowing to the right
 - You don't have a `Home` button on the about page to take us back to the homepage.
 - The about button in the navigation is notthe same colour with the one in the template.
 - The font size for these texts `youth and purity` and `Togo Baptist Church National Youth Conference` are not the same.
- The width of the text in the box and the date and even the header should be increased. That is why the date is breaking to another line

| Your Design |
|--------------|
|![baptist1](https://user-images.githubusercontent.com/62938735/190479949-c3363cee-cd22-4a4a-afde-0258bbabf7fc.gif)|

- [ ] Kindly implement the change requested in the image below [REQUIRED]
  
| Your Design |
  |-------------|
  |![image](https://user-images.githubusercontent.com/62938735/190483990-57344000-e9d6-4e24-be29-d4883680db29.png)|

- [ ] Kindly change the background color of the mobile navigation pop-up from. black to transparent [REQUIRED]


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- I noticed some amount of errors in spelling, grammar and case. For example, you started the title of our PR with lowercase and even the body of your PR. The same thing applies to some of your commit messages. You are expected to start a sentence in uppercase. I will suggest that you take note of this and download a tool like [Grammarly](https://www.grammarly.com) for your web browser and document editor. It will help a great deal in your writing. And I can. assure you that you have a great deal of writing to do on this journey 😉  [OPTIONAL]
- Kindly make your README more descriptive. For example, you can tell us how to clone and use your app. You can. follow this [link](https://github.com/fobadara/gcxchange) for ideas. [OPTIONAL]
- Kindly ensure your commit messages are in the imperative. You can. check [this](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/) out to know more [OPTIONAL].
- Your pull request title should also be in the imperative [OPTIONAL]
-  I noticed that you named the feature branch `dev`. The name of a feature branch is supposed to be a feature or an action. For example, navigation_bar or create_nav_bar.  Another reason you should. avoid using the name dev because it is a convention to name some of your branches dev or development when you are using Git-flow. You are currently using GitHub flow and will start using Git flow in react. Mixing it up could cause confusion[OPTIONAL]
- It is kind of odd that you have just two commits for an entire capstone project. Committing regularly will save you a lot of stress in the future. For example, after creating navigation you can commit. Then another when you create the homepage and so on [OPTIONAL]
- There is nothing you can do about it but in the future when you are using GitHub flow you are not expected to write any code or create a pull request n main or master everything should be done in a feature branch [OPTIONAL]
- Good job 👍🏽  using semantic tags but I believe there should be place for a main tagin your code [OPTIONAL]


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

hopeforkoami committed:
Update home and about desktop styles

hopeforkoami committed:
Update readme file

hopeforkoami committed:
Update style files

hopeforkoami committed:
Update index_desktop.css

hopeforkoami committed:
Remove partner section on mobile index

hopeforkoami committed:
Remarks correction

Taking into account all the required and optional comments made by the reviewer

hopeforkoami committed:
Update style.css

hopeforkoami committed:
Update index_desktop.css

hopeforkoami committed:
Update README.md

levy002 reviewed (changes requested):
## Hello @hopeforkoami, 
### I'm Levy your PR reviewer.

Good job so far! :+1: 
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

![](https://media.giphy.com/media/nYYtQdLqwwlTe3ccKh/giphy.gif)


## Suggested changes
_Check the comments under the review._

-  [ ] As requested by the previous reviewer, kindly make sure to make your mobile menu's background is transparent or the same color/pattern as the top section background.    
 
- [ ] Kindly make sure the logo is aligned correctly on the desktop version as the one in the provided design

  - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/190525048-8059945b-1c17-48c4-8ac6-a519b918a4e1.png)


  - ### Provided design
   ![image](https://user-images.githubusercontent.com/87197412/190525093-a8b440fd-2799-442e-909e-effd27d4dc90.png)


You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

hopeforkoami committed:
Remarks correction

Taking into account all the comments made by the reviewers
- align logo correctly on the desktop version as the one in the provided design
- mobile menu's background is transparent or the same color/pattern as the top section background

hopeforkoami committed:
Update video link

Update video link in the readme file

danielufeli reviewed (approved):
Hi @hopeforkoami ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![a](https://camo.githubusercontent.com/40c24648ada128030359495b9504e95985194c3f4bbeabf0485e740ecd253bcc/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6a4a5143327075565a70544d4f34765573302f67697068792e676966)

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
